docs/server/minecraft.md

```diff
@@ -36,6 +36,16 @@ Discord Server for support with Software developed by SpongePowered.
 **Invite**:  
 https://discord.gg/sponge
 
+----
+# Server Panels
+Discord server for administrative panels of Minecraft or other games
+
+#### :discord-verified: Pterodactyl
```

Comment:
Just put this under the other servers.
I don't see any reason as to why there should be a separate category just for one type so far.